pkg/components/components.go

```diff
@@ -358,6 +358,22 @@ func GetDeployment(version string, operatorVersion string, namespace string, rep
 }
 
 func GetRole(namespace string) *rbacv1.Role {
```

Comment:
at least some of the changes here are cosmetics which i personally against (it is clearer as it was imo)
and also it is not related to the PR
please remove them, we can create a follow-up PR and discuss them there

if any logical change is needed, lets add just that (maybe it is in a commit already, but i looked on the total diff)

Reply:
sure I'll move to a separate PR